module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -1877,14 +1877,12 @@ public function getConfig($func, $params = null)
     {
         if ($func == "Holds") {
             $holdsConfig = $this->config['Holds'] ?? [];
-            return [
-                "HMACKeys" => $holdsConfig['HMACKeys'] ?? "id:item_id",
-                "extraHoldFields"
-                    => $holdsConfig['extraHoldFields']
-                        ?? "comments:requiredByDate:pickUpLocation",
-                "defaultRequiredDate"
-                    => $holdsConfig['defaultRequiredDate'] ?? "0:1:0"
+            $defaults = [
+                'HMACKeys' => 'id:item_id',
+                'extraHoldFields' => 'comments:requiredByDate:pickUpLocation',
+                'defaultRequiredDate' => '0:1:0',
             ];
+            return $holdsConfig + $defaults;
```

Comment:
@demiankatz, does this actually mean `array_merge($holdsConfig, $defaults)`? 🤔

Reply:
@welblaud not exactly, see this: https://www.texelate.co.uk/blog/combining-arrays-using-plus-versus-array-merge-in-php

You could read this code like: Get holds config from ini file, if there is any mandatory key missing, add it from defaults (but not overwrite it - that's how the plus operator works here)

Reply:
Ok, got it, thanks for the explanation :)